### PR TITLE
fix bug

### PR DESCRIPTION
**Pull request by PauliusGrigaliunas**

### CODE REVIEW

File: TravelAgent/TravelAgent/DataContract/Responses/EmployeeTravelResponse.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+using TravelAgent.Data.Entities;
+
+namespace TravelAgent.DataContract.Responses
+{
+    public class EmployeeTravelResponse
+    {
+        public int Id { get; set; }
+        public Employee Employee { get; set; }
+        public Travel Travel { get; set; }
+    }
```
Comment (WarHatch) [inline TravelAgent/TravelAgent/DataContract/Responses/EmployeeTravelResponse.cs:15]: O čia neturėtų dar būt `bool confirm`?

File: TravelAgent/TravelAgent/Controllers/EmployeeTravelController.cs [hunk 2]
```diff
@@ -78,12 +79,17 @@ public async Task<ActionResult<Travel>> CreateEmployeeTravel([FromBody]CreateEmp
...
                     await _apartmentRepository.AddGuest(await _apartmentRepository.FindById(_office.OfficeApartment.Id),
                         _employeeTravel);
-                        
-                    return Ok(_employeeTravel);
+
+                    var employeeTravelResponse = new EmployeeTravelResponse
+                    {
+                        Id = _employeeTravel.Id,
+                        Employee = _employeeTravel.Employee
+                    };
```
Comment (WarHatch) [inline TravelAgent/TravelAgent/Controllers/EmployeeTravelController.cs]: What value is set for Travel?

**Review by WarHatch — CHANGES_REQUESTED** [review]
Unsure about stability

**Comment by WarHatch** [comment]
GetById and GetAll EmployeeTravel still return null for hotels and transports 😭

File: TravelAgent/TravelAgent/DataContract/Responses/EmployeeTravelResponse.cs (hunk 1 above)
Comment (WarHatch) [reply TravelAgent/TravelAgent/DataContract/Responses/EmployeeTravelResponse.cs:15]: @PauliusGrigaliunas last one remaining

Comment (PauliusGrigaliunas) [reply TravelAgent/TravelAgent/DataContract/Responses/EmployeeTravelResponse.cs:15]: I didn't use it, yet

**Review by WarHatch — APPROVED** [review]
https://media.giphy.com/media/l0HTYUmU67pLWv1a8/giphy.gif

**Pull request merged**